src/openms/include/OpenMS/KERNEL/MRMTransitionGroup.h

```diff
@@ -335,6 +335,12 @@ namespace OpenMS
         }
       }
 
+      for (typename std::vector<SpectrumType>::iterator pr_it = precursor_chromatograms_.begin(); pr_it != precursor_chromatograms_.end(); ++pr_it)
```

Comment:
We need to copy the precursor chromatograms to the subset MRMTransitionGroup if transition-level scoring is enabled.